lib/ex_unit/lib/ex_unit/doc_test.ex

```diff
@@ -61,12 +61,16 @@ defmodule ExUnit.DocTest do
       2
 
   If you don't want to assert for every result in a doctest, you can omit
-  the result:
+  the result in the middle of expressions:
```

Comment:
```suggestion
  the result in between of expressions:
```
There's a subtle different but "in the middle" reads more like "within"